formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -428,6 +448,64 @@ internal class CborDecoder(private val input: ByteArrayInput) {
         return result
     }
 
+    fun skipElement() {
+        val lengthStack = mutableListOf<Int>()
```

Comment:
Is there a multiplatform-friendly dynamically sizing stack I could use in its place? Or would I have to roll my own?

If needed I could try to build a basic stack implementation to avoid the auto-boxing but I'm not confident my implementation would be optimal.

Reply:
Well, there's an `IntArrayBuilder` for deserializing IntArrays, but it is hidden for now. I think we can go with MutableList<Int> until we have a clear benchmarks that show if this is a real problem. @qwwdfsad WDYT?